src/dune_pkg/local_package.ml

```diff
@@ -119,12 +119,12 @@ let of_package (t : Dune_lang.Package.t) =
     let opam_file =
       Opam_file.read_from_string_exn ~contents:opam_file_string (Path.source file)
     in
-    let convert_filtered_formula = Package_dependency.list_of_opam_filtered_formula loc in
+    let convert_filtered_formula = Package_dependency.list_of_opam_disjunction loc in
```

Comment:
Refactoring idea: Maybe this identifier is not useful anymore? It's just contributes a single curried argument. Feel free to ignore if you think it helps readability.

Reply:
Thanks, I agree, I changed it :P (it's one line longer due to formatting, but it reads better with the rest of the code without `convert_filtered_formula`)